README.md

```diff
@@ -132,6 +132,38 @@ In the summer of 2024, this process took less than 5 hours to bin all 170 GB of
 
 ### Mapping
 
+#### Required Environment Variables
```

Comment:
Can you add some explanation about what do we require this?
Such as:
* ipinfo.io is used to achieve x and y because we need z and w
* Same for the salt.

Reply:
OK good point

Reply:
> ipinfo.io is used to achieve x and y because we need z and w

`ipinfo.io` is used to extract the general geographic region (rather than exact physical address) for 'anonymized' geo statistics

DM'd you on Slack about the salt

Reply:
@CodyCBakerPhD Thank you, I incorporated  your comment in [becfc2c](https://github.com/catalystneuro/dandi_s3_log_parser/pull/71/commits/becfc2c5c0564127b6e0dade95edacd5fc63dc8d)
@h-mayorquin let me know what you think